manta-trusted-setup/src/ceremony/coordinator.rs

```diff
@@ -150,41 +123,63 @@ where
         self.queue.position(participant)
     }
 
-    /// Updates the MPC state and challenge using client's contribution. If the contribution is valid,
-    /// the participant will be removed from the waiting queue, and cannot participate in this ceremony
-    /// again.
+    /// Updates the expired lock by reducing the priority of it's participant and setting it's
+    /// contained value to the new front of the queue. The previous participant in the lock is
+    /// returned.
+    #[inline]
+    fn update_expired_lock(&mut self) -> Option<ParticipantIdentifier<C>> {
```

Comment:
@bhgomes I am not sure if `lock` is the best name to describe `Timed<Option<ParticipantIdentifier>>`. This is because `Coordinator` itself is also guarded by a "lock" which has different meaning.

I don't mind if we make the name more verbose, something like `current_paritipant`

Reply:
@tsunrise yeah lets call it `participant_lock` 👍